src/data_manager/collectors/persistence.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import annotations
+
+import hashlib
+import re
+from pathlib import Path
+from typing import Dict
+
+import yaml
+
+from src.data_manager.collectors.scrapers.scraped_resource import \
+    ScrapedResource
+from src.data_manager.collectors.tickets.ticket_resource import TicketResource
+from src.utils.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+class PersistenceService:
+    """Shared filesystem persistence for collected resources."""
+
+    def __init__(self, data_path: Path | str) -> None:
+        self.data_path = Path(data_path)
+        self.sources_path = self.data_path / "sources.yml"
+        self.websites_dir = self.data_path / "websites"
+        self.git_dir = self.data_path / "git"
+        self.tickets_dir = self.data_path / "tickets"
+        self.tickets_index_path = self.data_path / "tickets.yml"
```

Comment:
I am having a hard time understanding: are sources.yml and tickets.yml the two different databases (one for links, one for tickets) for the mapping of useful metadata for each document?